app/build.gradle

```diff
@@ -59,7 +59,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.5.1'
     implementation 'com.google.android.material:material:1.7.0'
     implementation "androidx.compose.ui:ui:$compose_version"
-    implementation "androidx.compose.material:material:1.3.1"
+    implementation "androidx.compose.material:material:1.4.0"
```

Comment:
Could be extracted to a variable so it can be used for both app and library?

Reply:
Done! However, the issue happens when there is a different version between the library and one of our apps.